static/app/views/replays/replayTable/index.tsx

```diff
@@ -140,13 +143,124 @@ function ReplayTable({
   );
 }
 
+export function CardReplayTable({
+  fetchError,
+  isFetching,
+  replays,
+  sort,
+  visibleColumns,
+}: Props) {
+  const routes = useRoutes();
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const tableHeaders = visibleColumns
+    .filter(Boolean)
+    .map(column => (
+      <HeaderCell key={column} column={column} sort={sort} headersSortable={false} />
+    ));
+
+  if (fetchError && !isFetching) {
+    return (
+      <StyledPanelTable
+        headers={tableHeaders}
+        isLoading={false}
+        visibleColumns={visibleColumns}
+        data-test-id="card-replay-table"
+      >
+        <StyledAlert type="error" showIcon>
+          {typeof fetchError === 'string'
+            ? fetchError
+            : t(
+                'Sorry, the list of replays could not be loaded. This could be due to invalid search parameters or an internal systems error.'
+              )}
+        </StyledAlert>
+      </StyledPanelTable>
+    );
+  }
+
+  const referrer = getRouteStringFromRoutes(routes);
+  const eventView = EventView.fromLocation(location);
```

Comment:
I think if we were to refactor this `eventView` variable so that it were a prop then we could share a lot more of the ReplayTable and CardReplayTable code.

Looking at the other code that could be shared in this top section of the ReplayTable and CardReplayTable we've got:
- `const tableHeaders` the only difference is `headersSortable`, we could accept a prop at the ReplayTable level, and then pass it into here
- `if fetchError && !isFetching ...` that all looks the same, especially for now no need to make changes
- `const referrer = ...` will probably stay the same.

So a couple props: `function ReplayTable({eventView, sortableColumns, ... snip ...}: Props) {`
and then of course the `switch` statement has to have cases for all possible columns. So it's one giant switch and time we render a table, some set of columns will be used.

Also if